packages/cloud-cognitive/src/components/EditFullPage/EditFullPage.stories.js

```diff
@@ -0,0 +1,452 @@
+/**
+ * Copyright IBM Corp. 2021, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import React, { useState } from 'react';
+import { carbon } from '../../settings';
+import {
+  getStoryTitle,
+  prepareStory,
+} from '../../global/js/utils/story-helper';
+import { action } from '@storybook/addon-actions';
+import { CreateFullPage } from '../CreateFullPage';
+import { CreateFullPageStep } from '../CreateFullPage/CreateFullPageStep';
```

Comment:
We should make sure we’re using EditFullPage since we created a separate component for it.

Reply:
As per the instructions, we are not creating new components. We are creating only stories from existing components. Here these imports are required for creating story for EditFullPage.